src/test/java/org/openrewrite/java/migrate/lang/var/UseVarForGenericsConstructorsTest.java

```diff
@@ -357,6 +357,63 @@ void m() {
             );
         }
 
+        //see https://app.moderne.io/results/CuzcK/details/eyJfX3R5cGVuYW1lIjoiR2l0SHViUmVwb3NpdG9yeSIsIm9yaWdpbiI6ImdpdGh1Yi5jb20iLCJwYXRoIjoic3ByaW5nLXByb2plY3RzL3NwcmluZy12YXVsdCIsImJyYW5jaCI6Im1haW4ifQ==?referrer=%2Fresults%2FCuzcK%3Ffilter%3DeyJpdGVtcyI6W3siZmllbGQiOiJzdGF0dXMiLCJvcGVyYXRvciI6Im5vdCIsImlkIjo0Nzc3MiwidmFsdWUiOiJGSU5JU0hFRCJ9XSwibG9naWNPcGVyYXRvciI6ImFuZCIsInF1aWNrRmlsdGVyVmFsdWVzIjpbXSwicXVpY2tGaWx0ZXJMb2dpY09wZXJhdG9yIjoiYW5kIn0%25253D
```

Comment:
These URLs only last as long as the worker that holds them, so while I appreciate the content for review, it's likely to break soon, so let's remove it from the code.
```suggestion
```